FirstProject/new-evaluation.html

```diff
@@ -0,0 +1,536 @@
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+
+                    </li>
+                </ul>
+            </fieldset>
+        </form>
+
+        <form class="language_form" style="text-align:center;">
+            <fieldset class="field_section" style="display:inline;">
+                <legend>CSS</legend>
+                <ul style="display: inline-flex; text-align: right">
+                    <li>
+                        Usage <select>
+                        <option selected disabled>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                        Selectors <select>
+                        <option selected disabled>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                    </li>
+
+                    <li>
+                        Box modeling <select>
+                        <option selected disabled>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                        Styling <select>
+                        <option selected disabled>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                    </li>
+
+                    <li>
+                        CSS 3.0 <select>
+                        <option selected disabled>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                        Dynamic Stylesheets <select>
+                        <option selected disabled>Evaluation</option>
+                        <option>0</option>
+                        <option>1</option>
+                        <option>2</option>
+                        <option>3</option>
+                    </select> <br>
+                    </li>
+                </ul>
+            </fieldset>
+        </form>
+
+        <form class="language_form" style="text-align:center;">
+            <fieldset class="field_section" style="display:inline;">
+                <legend>Javascript</legend>
+                <ul style="display: inline-flex; text-align: right">
+                    <li>
+                        Data types & variables <select>
```

Comment:
More and more of these seemingly erroneous `&`... maybe we should look into what's up?